docs/chapter4-payload-handoff-format.rst

```diff
@@ -29,9 +29,833 @@ architecture, either 32 or 64 bits.
+                                                      so it is the parent's cell count of address and size used.
+   ranges            SD          prop-encoded-array   Refer to here for generic definition: :ref:`sp_ranges`
+                                 or empty             
+                                                      For this node, ranges data is in the form of PCI address, CPU address, PCI size. The cells of PCI address is the value of #address-cells in PCI
+                                                      node (3). The cells of CPU address is the value of #address-cells in the parent node. The cells of PCI size is the value of #size-cells in PCI
+                                                      node (2).
+
+                                                      The data is used to define IO, prefetchable/non-prefetchable MMIO resources that can be utilized for the root bridge which can be assigned to
+                                                      downstream devices.
+                                                      
+                                                      The ranges include those MMIO or IO which are already assigned to downstream devices and available for future new devices.
+
+                                                      If the property is defined with an <empty> value, it specifies that the parent and child address space is identical, and no address translation is required.
+
+                                                      If the property is not present in a bus node, it is assumed that no mapping exists between children of the node and the parent address space.
+   dma-ranges        O           prop-encoded-array   Refer to here for generic definition: :ref:`sp_dma-ranges`
+                                 or empty             
+                                                      Define the DMA address width capability of the root bridge. If only below 4GB region was defined, the payload may assume above 4GB DMA is not
+                                                      supported.
+   ================= =========== ==================== ======================================================================================================================================================================================================================================================================================================================================================================================================================================
+
+An example showing multiple top-level PCI nodes each with its own bus range:
+
+.. code-block:: dts
+
+   
+   //1st pci-rb in its separate segment of PCIEXBAR 0xC0000000
+   pci-rb0@c0000000 {
+      compatible = "pci";
+      #address-cells = <3>;
+      #size-cells = <2>;
+      bus-range = <0x01 0xdf>;
+      reg = <0x0 0xc0000000 0x1 0x0000000>; // whole ECAM region in this root bridge
```

Comment:
Can you put the comment on its own line before this one? It overflows in the PDF

Reply:
i also saw in the pdf, i think you are referring to the other comment line, not this one right? anyway, resolved.